src/main/java/me/zilzu/mycoupon/application/service/CouponRepository.java

```diff
@@ -27,4 +27,12 @@ public Coupon retrieve(String id) {
         
         return coupon;
     }
+
+    public Long getAllCoupon() {
+        return (long) database.size();
+    }
```

Comment:
```suggestion
    public Long getAllCouponSize() {
        return (long) database.size();
    }
```

이것도 베스트는 아닌거같지만, `getAllCoupon()`이라 함은 List<Coupon> 을 반환할 것만 같음.